app/main.py

```diff
@@ -2,0 +1,15 @@
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves_list: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in elves_list]
+
+
+def feast_of_the_dwarves(dwarves_list: list[Dwarf]) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarves_list]
```

Comment:
Creating a list with a comprehension for side effects is unnecessary. Use a simple loop instead: `for dwarf in dwarves_list: dwarf.eat_favourite_dish()`.